### PR TITLE
regeditor: Add ItemType to preinstall entry

### PR DESCRIPTION
**Pull request by Dabombber**
Otherwise the user is prompted to specify the type.

### CODE REVIEW

File: bucket/regeditor.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     },
     "pre_install": [
         "if (!(Test-Path $persist_dir\\OORegEdt.ini)) {",
-        "   New-Item $dir\\OORegEdt.ini | Out-Null",
+        "   New-Item -ItemType file $dir\\OORegEdt.ini | Out-Null",
```
Comment (Ash258) [inline bucket/regeditor.json]: Positional parameters should go first.
```suggestion
        "   New-Item \"$dir\\OORegEdt.ini\" -ItemType File  | Out-Null",
```

File: bucket/regeditor.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
     },
     "pre_install": [
         "if (!(Test-Path $persist_dir\\OORegEdt.ini)) {",
```
Comment (Ash258) [inline bucket/regeditor.json]:
```suggestion
        "if (!(Test-Path \"$persist_dir\\OORegEdt.ini\")) {",
```

**Comment by niheaven** [comment]
What's your Powershell version? In PS 5 and 6, `New-Item` will create file by default.

**Comment by Dabombber** [comment]
> 
> 
> What's your Powershell version? In PS 5 and 6, `New-Item` will create file by default.

It was 3, which was the minimum recommended. I guess it might not be enough any more lukesampson/scoop#2402.

**Pull request merged**